.env.development

```diff
@@ -1,8 +1,12 @@
+ABOUT_US_URL=
 ACCESS_TOKEN_COOKIE_NAME=edx-jwt-cookie-header-payload
+ACCESSIBILITY_URL=
```

Comment:
I think the accessibility is a mandatory page for a website to make it compliant with legalities. I think we should have a default fallback value for this. e.g. how we are making sure in StudioFooter. By default, we are sending the user to STUDIO_URL/accessibility.